capstone-overview/logistics.md

```diff
@@ -0,0 +1,12 @@
+## Capstone Logistics to Review
+
+1. **Days off:** The only holiday during capstone period is President's Day on February 15. Instructors will not be available to support on that day.
+1. **Standup Groups:** PMs and Standup groups will be assigned on Tuesday January 26. Standups should occur daily starting Wednesday January 27. Each group is able to choose its own time to meet together.
+1. **Staff & TA Support:** In addition to your Standup support groups and your assigned PM, Instruction Staff and TAs are happy to support with your projects.  Checkout the [Capstone Support Form spreadsheet](https://docs.google.com/spreadsheets/d/1QKlTPp5TX7YpZC3hO091VSd_M_eklNHagkJQ164bIic/edit?usp=sharing) for information of who could be most helpful with the technologies you're using.
+1. **Standup Form:** There will be a stand-up form you must fill out every day before your group's standup. Your PM will be checking that it was submitted. This helps you and your PM track your progress and helps ensure you're prepared to give a succinct, clear standup update to your group.
+1. **Week One:** The first week (Jan 25 - 29) is intended to be primarily learning focused. We encourage people to spend much of that time learning and experimenting with the languages and frameworks they plan to use. Not much progress on building features for the project is necessarily expected during this first week.
+1. **Recurring things:**
+    - PMs will meet with each individual weekly
```

Comment:
I'm trying to figure out the words to say that 1:1s can optionally start Thursday 1/28, but I'm having trouble wording it

Reply:
Yeah I ran into the same problem. Haha. 
I decided that's just something we can explain to them in a timely manner, like on Tuesday or Wed when it feels more relevant

Reply:
Hope that works! Thanks for looking again